test/src/e2e_vm_tests/test_programs/should_pass/language/logging/json_abi_oracle.json

```diff
@@ -1,79 +1,56 @@
 {
   "configurables": [],
+  "encoding": "1",
   "functions": [
     {
       "attributes": null,
       "inputs": [],
       "name": "main",
       "output": {
         "name": "",
-        "type": 14,
+        "type": 13,
         "typeArguments": null
       }
     }
   ],
   "loggedTypes": [
     {
-      "logId": 0,
+      "logId": 4579537983717831593,
       "loggedType": {
         "name": "",
-        "type": 7,
+        "type": 6,
         "typeArguments": []
       }
     },
     {
-      "logId": 1,
+      "logId": 16566583104751091389,
       "loggedType": {
         "name": "",
-        "type": 8,
+        "type": 7,
         "typeArguments": [
           {
             "name": "",
-            "type": 14,
+            "type": 13,
             "typeArguments": null
           }
         ]
       }
     },
     {
-      "logId": 2,
-      "loggedType": {
-        "name": "",
-        "type": 1,
-        "typeArguments": []
-      }
-    },
-    {
-      "logId": 3,
+      "logId": 5087777005172090899,
```

Comment:
@kayagokalp not sure why this didn't get caught by tests, but these are still non-str values

Reply:
That is interesting, investigating now. Maybe during rebasing I mixed up things and modified the oracle invalidly somehow. But the fact that `output` file is generated as using strings tells me the code itself is working correctly. 

It is weird that this is not caught by the test suite.

Reply:
Not sure how this is not caught. it seems like test suite is doing string comparison between oracle and the output

https://github.com/FuelLabs/sway/blob/2ec4332c550e3bad4cc795b189342f1d52b14ebf/test/src/e2e_vm_tests/harness.rs#L398